runtime/compiler/rpc/raw/J9Server.cpp

```diff
@@ -53,6 +56,7 @@ J9ServerStream::J9ServerStream(int connfd, BIO *ssl, uint32_t timeout)
    : J9Stream(),
    _msTimeout(timeout)
    {
+   _numConnectionsOpened++;
```

Comment:
Let's move this at the very end of this function. If the constructor throws in initStream we don't want to increment the number of connections open.

Reply:
I intentionally did this so that we could see how many J9ServerStream constructions were attempted. But i guess the purpose of this is to check memory/connection leaks, i should move it to the end of the function.